Dockerfile

```diff
@@ -0,0 +1,48 @@
+# openhab image for x86
+FROM ubuntu
+
+RUN \
+          apt-get update && \
+          apt-get install -y wget unzip software-properties-common && \      
+          rm -rf /var/lib/apt/lists/*
+
+# Install dockerize
+RUN wget -nv https://github.com/jwilder/dockerize/releases/download/v0.2.0/dockerize-linux-amd64-v0.2.0.tar.gz &&\
+    tar -C /usr/local/bin -xzvf dockerize-linux-amd64-v0.2.0.tar.gz &&\
+    rm dockerize-linux-amd64-v0.2.0.tar.gz
+
+# Install Oracle Java
+RUN \
+          echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | debconf-set-selections && \
+          add-apt-repository -y ppa:webupd8team/java && \
+          apt-get update && \
+          apt-get install -y oracle-java8-installer && \      
+          rm -rf /var/lib/apt/lists/* && \
+          rm -rf /var/cache/oracle-jdk8-installer
+
+ENV DOWNLOAD_URL="https://openhab.ci.cloudbees.com/job/openHAB-Distribution/lastSuccessfulBuild/artifact/distributions/openhab-online/target/openhab-online-2.0.0-SNAPSHOT.zip" \
+    APPDIR="/openhab"
+
+WORKDIR ${APPDIR}
+
+RUN wget -nv -O /tmp/openhab.zip ${DOWNLOAD_URL} &&\
+    unzip -q /tmp/openhab.zip -d ${APPDIR}
+# -q
+#RUN rm /tmp/openhab.zip
+
+# Add openhab user
+RUN adduser --disabled-password --gecos '' openhab &&\
+    adduser openhab sudo &&\
+    echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers.d/openhab
+
+RUN mv ${APPDIR}/userdata /userdata && ln -s /userdata ${APPDIR}/userdata
+RUN mv ${APPDIR}/conf /config && ln -s /config ${APPDIR}/conf
+
+RUN chown -R openhab:openhab ${APPDIR} /config /userdata
+
+USER openhab
+RUN mkdir -p ${APPDIR}/userdata/logs && touch ${APPDIR}/userdata/logs/openhab.log
+# Expose volume with configuration and userdata dir
+VOLUME /config /userdata
+EXPOSE 8080 8443 5555
+CMD dockerize -stdout ${APPDIR}/userdata/logs/openhab.log ${APPDIR}/start.sh server
```

Comment:
It might make sense to have $DEBUG Environment variable to start with debug instead os server